pxtlib/emitter/cloud.ts

```diff
@@ -34,26 +34,26 @@ namespace pxt.Cloud {
             && !BrowserUtils.isLocalHost() && !!pxt.webConfig.cdnUrl
     }
 
-    export function cdnApiUrl(url: string) {
+    export function cdnApiUrl(url: string, hourly?: boolean) {
         url = url.replace(/^\//, '');
         if (!useCdnApi())
             return apiRoot + url;
 
         const d = new Date()
-        const timestamp = d.getUTCFullYear() + ("0" + (d.getUTCMonth() + 1)).slice(-2) + ("0" + d.getUTCDate()).slice(-2)
+        const timestamp = d.getUTCFullYear() + ("0" + (d.getUTCMonth() + 1)).slice(-2) + ("0" + d.getUTCDate()).slice(-2);
         if (url.indexOf("?") < 0)
             url += "?"
         else
             url += "&"
-        url += "cdn=" + timestamp
+        url += `cdn=${timestamp}${hourly ? d.getUTCHours() : ""}`;
```

Comment:
Hmm. I am not sure this is how it works. Backend looks for timestamp https://github.com/microsoft/pxt-backend/blob/e952a311066087dcbb42c872a26bd8504bff1d94/node/src/main.ts#L104

Reply:
Ahh you're right, I misunderstood the comparison on the timestamp; closing for now and will get a better fix then